flake.nix

```diff
@@ -35,5 +35,31 @@
                 };
             };
 
+            # Adding the new NixOS configuration for "ihp-app"
+            flake.nixosConfigurations."ihp-app" = nixpkgs.lib.nixosSystem {
+                system = "x86_64-linux";
+                specialArgs = inputs;
+                modules = [
+                    "${nixpkgs}/nixos/modules/virtualisation/amazon-image.nix"
+                    ihp.nixosModules.appWithPostgres
+                    ({ ... }: {
+                        security.acme.defaults.email = "no-reply@tpp-qa.gizra.site";
+
+                        services.ihp = {
+                            domain = "tpp-qa.gizra.site";
+                            migrations = ./Application/Migration;
+                            schema = ./Application/Schema.sql;
+                            fixtures = ./Application/Fixtures.sql;
+                            sessionSecret = "M$cmzMCEx7xfL-5_q6%9cpve_0BAd5BbDaOtzhv7";
```

Comment:
The session secret should not be public, anyone with knowledge of the `sessionSecret` can write to the session cookie (e.g. override the logged in user or something else)

Reply:
That's right, this is a dummy PR that we will throw away, we do not intend to deploy this project. Also we won't use this secret elsewhere.

Reply:
@mpscholten Yeah, we've done it so you'd have access to the code 😄 
We can also invite you to the dummy AWS instance - so you'd have full visibility of the problem.